CMakeLists.txt

```diff
@@ -281,45 +276,11 @@ set(CMAKE_INSTALL_NAME_DIR
 # enable 24 bit receiving path
 if (RX_SAMPLE_24BIT)
     message(STATUS "Compiling for 24 bit Rx DSP chain")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DSDR_RX_SAMPLE_24BIT")
+    add_compile_definitions(SDR_RX_SAMPLE_24BIT)
 else()
     message(STATUS "Compiling for 16 bit Rx DSP chain")
 endif()
 
-if (SANITIZE_ADDRESS)
-    message(STATUS "Activate address sanitization")
-    if(MSVC)
-      set(ASAN_LIB_ARCH ${MSVC_CXX_ARCHITECTURE_ID})
-      string(TOLOWER ${ASAN_LIB_ARCH} ASAN_LIB_ARCH)
-      if(ASAN_LIB_ARCH STREQUAL "x86")
-        set(ASAN_LIB_ARCH "i386")
-      elseif(ASAN_LIB_ARCH STREQUAL "x64")
-        set(ASAN_LIB_ARCH "x86_64")
-      endif()
-      add_compile_options(/fsanitize=address)
-      link_libraries(clang_rt.asan_dynamic-${ASAN_LIB_ARCH} clang_rt.asan_dynamic_runtime_thunk-${ASAN_LIB_ARCH})
-      add_link_options(/wholearchive:clang_rt.asan_dynamic_runtime_thunk-${ASAN_LIB_ARCH}.lib)
-    else()
-      add_compile_options(-fsanitize=address -fno-omit-frame-pointer -g)
-      add_link_options(-fsanitize=address)
-    endif()
-endif()
-
-# set compiler
-include(FindCompiler)
```

Comment:
Why did you remove cmake/Modules/FindCompiler.cmake. How can I tell between GCC and CLANG? This is important in other parts of the software!

Reply:
Because it was duplicated.

Reply:
Indeed I don't know why the code of FindCompiler landed in FindCPUFlags bet they could be kept separate as they do not relate logically to each other. And I never use the SANITIZE_ADDRESS so I don't mind removing it.